pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -99,6 +94,13 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
     return {
       templateUrl: 'teacher-dashboard/nav',
       controller: function($scope) {
+        // Show projects tab in the section nav if any of the following are true:
+        // 1. CDO.section_projects is set
+        // 2. DCDO.section_projects is set
+        // 3. sectionProjects url param is present
+        // 4. the projects tab is specified in the url
+        // The projects tab is accessible by direct url regardless of settings 1-3 above.
+        $scope.showSectionProjectsTab = #{show_section_projects} || $scope.tab === 'projects';
```

Comment:
Nope, this change was incidental. Good point. I've moved the ruby to the beginning of the file.